configure

```diff
@@ -159,6 +159,9 @@ while [ $# -ne 0 ]; do
     --show-time-report)
       append_cache_entry VAST_ENABLE_TIME_REPORT BOOL yes
       ;;
+    --show-time-trace)
+      append_cache_entry VAST_ENABLE_TIME_TRACE BOOL yes
+      ;;
```

Comment:
Please also update the usage help.

Reply:
Such a nuisance that we need to mirror this manually from CMake. I always forget this. Will do.

Reply:
Feel free to write shell function that generates the help automatically. 😉

Reply:
I'd rather get rid of the configure script entirely. It's an antipattern imo.